prebid-server/endpoints/openrtb2/pbs-endpoint-auction.md

```diff
@@ -150,15 +185,33 @@ How long an item is stored in Prebid Cache is determined by this hunt path:
 4. account config: {banner,video}-cache-ttl
 5. global config: cache.{banner,video}-ttl-seconds
 
-#### Privacy fields
+#### Timeout
+
+The OpenRTB 2.5 `imp[].exp` field defines how long Prebid Server has to process the request
+before the client will stop waiting.
+
+This field is used in different ways by PBS-Go and PBS-Java:
+
+##### PBS-Go
```

Comment:
Until PBS-Go implements the tmax issue, the behavior is:
- if tmax is not specified or is 0, use the host configured default value (auction_timeout_ms.default)
- if tmax is over the host defined max, cap it to the host defined max (auction_timeout_ms.max)